lib/jsdom/vm-shim.js

```diff
@@ -3,45 +3,24 @@
 const acorn = require("acorn");
 const findGlobals = require("acorn-globals");
 const escodegen = require("escodegen");
+const jsGlobals = require("./browser/js-globals.json");
 
 // We can't use the default browserify vm shim because it doesn't work in a web worker.
 
-// From ES spec table of contents. Also, don't forget the Annex B additions.
-// If someone feels ambitious maybe make this into an npm package.
-const builtInConsts = ["Infinity", "NaN", "undefined"];
-const otherBuiltIns = [
-  "isFinite", "isNaN", "parseFloat", "parseInt", "decodeURI", "decodeURIComponent",
-  "encodeURI", "encodeURIComponent", "Array", "ArrayBuffer", "Boolean", "DataView", "Date", "Error", "EvalError",
-  "Float32Array", "Float64Array", "Function", "Int8Array", "Int16Array", "Int32Array", "Map", "Number", "Object",
-  "Proxy", "Promise", "RangeError", "ReferenceError", "RegExp", "Set", "String", "Symbol", "SyntaxError", "TypeError",
-  "Uint8Array", "Uint8ClampedArray", "Uint16Array", "Uint32Array", "URIError", "WeakMap", "WeakSet", "JSON", "Math",
-  "Reflect", "escape", "unescape"
-];
+const jsGlobalEntriesToInstall = Object.entries(jsGlobals).filter(([name]) => name !== "eval" && name in global);
```

Comment:
Why skip eval? Maybe add a comment?

Reply:
This is because `eval` is set to a custom function: <https://github.com/jsdom/jsdom/blob/713f97aeebf63e959f9682bd9fe5b8fd56b80cf7/lib/jsdom/vm-shim.js#L28-L30>